ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource

```diff
@@ -16,7 +16,8 @@ ${ACTIONS_BTN_XP}=    xpath=//div/button[@aria-label="Actions"]
 ${DELETE_ACTION_BTN_XP}=  xpath=//div/ul/li/button[text()="Delete project"]
 ${EDIT_ACTION_BTN_XP}=    xpath=//div/ul/li/button[text()="Edit project"]
 ${SPAWNER_LINK}=        xpath=//*[text()="Launch Jupyter"]
-
+${EXISTING_DATA_CONNECTION_BTN_XP}=    xpath=//*[@id='existing-data-connection-type-radio']/parent::*//button[1]
```

Comment:
Does this button have a better locator? numerical indexes are prone to breaking in my experience.
You can also use /../ instead of /parent::*/

Reply:
Removed